server/tools/jgroups.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I don't agree here. 

I can imagine use cases where you will need multiple protocols for discovery (as we discussed a few days ago). Or maybe you will need to adjust `FD_*` protocol parameters with discovery timeouts (this is often the case when using `KUBE_PING` that polls Kubernetes API in a loop). In such case you will need to provide your own cli file that does it. As it is, it's super easy, just extend our image and drop a file into `/opt/jboss/tools/cli/jgroups/discovery`. If we removed this mechanism, you would need to drop a cli file and then copy-paste the `jgroups.sh` file. Now if you copy that file, you will need to maintain it manually by syncing manually changes we made in it with your own copy. That's a maintenance pain that is very easy to avoid with my proposed approach.

However, you might have a good point that this behavior should be described in the README. I'll add it.

Reply:
Added necessary manual entries.